assets/sass/common/shared/o-forms/_o-form.scss

```diff
@@ -441,7 +441,6 @@
 %custom-checkbox-label {
   display: block;
   position: relative;
-  line-height: 16px;
```

Comment:
Looks like even okta-ui/courage has this setting. Can we double check with design team before making changes?

Reply:
The effective change is `line-height: 18.2px` after removing this. It seems like this code was added years ago and it probably went unnoticed. I have removed it since it fixes the misalignment issue, will change it if @pradeepdewda-okta thinks otherwise..

Since the checkbox gets created from backend directly in my case, i'll have to add the class somewhere in the uiSchema before it gets created. It makes sense to me to have a generic fix for all checkboxes in SIW in this case.

Reply:
@taylorlaubach-okta what are your thoughts ?

Reply:
Changed the styling to target only v2 - remember me checkbox.

Reply:
we wouldn't like one-off in general. If what Pradeep found is an issue and ux team agreed, we could fix the infobox common style as well as in okta-ui (could be separated ticket)

Reply:
@pradeepdewda-okta Thanks for looping me in! I agree with you--the old alignment looks bad and the fix looks correct.

I'm okay with fixing the issue anywhere it exists. Just want to make sure that we don't break correct alignment anywhere. (For example, if some checkboxes need the `line-height: 16px` style in order to be correctly aligned and removing it breaks that for them.)

Reply:
Thanks @taylorlaubach-okta.!! @anipendakur-okta it's your call to fix it in this PR or create a new one and provide the fix there. Any case it should not hold you to merge this PR.

Reply:
It makes sense to address it as a separate JIRA since this involves testing all radios and checkboxes on sign-in-widget. I opened a new JIRA to address it - https://oktainc.atlassian.net/browse/OKTA-303139

Reply:
@anipendakur-okta Thanks for doing that! Let me know if you need anything from me.